### PR TITLE
Format Python script using Black formatter

### LINKED ISSUE #1200: Format Python scripts using Black
Hi there,

What if we format the Python scripts `scripts/sort_contibutors/main.py` using [Black](https://github.com/psf/black)? The goal of formatting is to increase readability of the code (as we follow the standardized format on writing the code).

If OK, I'll create a PR for this. So far, I only found one Python script on this repo.

### PR DESCRIPTION
**Pull request by za**
### What kind of changes does this PR include?

-   [ ] Fixes or refactors
-   [ ] A new challenge
-   [ ] Additional documentation
-   [x] Something else

#### Description

Format Python script using Black formatter so that the style is consistent. Hopefully this will make the code more readable and attract more contributors.

### Relations

Closes #1200 

### References

https://github.com/psf/black

### Checklist:

-   [x] All the contributions made are solely the work of me and my co-authors
-   [x] I tested the changes in this PR (if applicable)
-   [x] I added unit tests to ensure my change works (when change in Java or on front-end code)
-   [x] I added UI tests to ensure my UI changes work (when change in the overall UI, not needed if just adding a challenge)
-   [x] The PR passes pre-commit hooks and automated tests

### CODE REVIEW

**Review by commjoen — CHANGES_REQUESTED** [review]
Hi @za ,
Can you please take a look on how we can have the formatting done automatically using precommit? Thanks!

**Comment by za** [comment]
Hi @commjoen ok, I'll have a look.

**Comment by za** [comment]
> Hi @za , Can you please take a look on how we can have the formatting done automatically using precommit? Thanks!

Hi @commjoen I've added `black.yml` and updated `pre-commit.yml` file. Can you check?

File: .github/workflows/pre-commit.yml [hunk 1]
```diff
@@ -23,6 +23,12 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: "3.9"
+      - name: Format python code with black
+        uses: psf/black@stable
+        with:
+          options: "--check --verbose"
+          src: "./scripts"
+          version: "~= 22.0"
```
Comment (commjoen) [inline .github/workflows/pre-commit.yml:31]: Hi @za ,
Can you remove this and update the precommit config instead please? This is  the GitHub workflow.

Comment (commjoen) [reply .github/workflows/pre-commit.yml:31]: See https://github.com/OWASP/wrongsecrets/blob/master/.pre-commit-config.yaml

Comment (za) [reply .github/workflows/pre-commit.yml]: > Hi @za , Can you remove this and update the precommit config instead please? This is the GitHub workflow.

Removed.

**Comment by commjoen** [comment]
Something seems to fail when using the new pre-commit hook:

```sh
[26/01 4:42:29] ➜  sort_contibutors git:(python-black-formatted) ✗ pre-commit install
pre-commit installed at .git/hooks/pre-commit
[26/01 4:42:34] ➜  sort_contibutors git:(python-black-formatted) ✗ git commit        
[INFO] Initializing environment for https://github.com/psf/black.
An unexpected error has occurred: CalledProcessError: command: ('/usr/local/opt/git/libexec/git-core/git', 'checkout', '22.6b0')
return code: 1
expected return code: 0
stdout: (none)
stderr:
    error: pathspec '22.6b0' did not match any file(s) known to git
    
Check the log at /Users/razr/.cache/pre-commit/pre-commit.log
[26/01 4:42:38] ➜  sort_contibutors git:(python-black-formatted) ✗ git status
On branch python-black-formatted
Your branch is up to date with 'origin/python-black-formatted'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   main.py

```

**Comment by za** [comment]
> Something seems to fail when using the new pre-commit hook, can you please fix the black integration :) ? Then I am very happy to continue :) .
> 
> ```shell
> [26/01 4:42:29] ➜  sort_contibutors git:(python-black-formatted) ✗ pre-commit install
> pre-commit installed at .git/hooks/pre-commit
> [26/01 4:42:34] ➜  sort_contibutors git:(python-black-formatted) ✗ git commit        
> [INFO] Initializing environment for https://github.com/psf/black.
> An unexpected error has occurred: CalledProcessError: command: ('/usr/local/opt/git/libexec/git-core/git', 'checkout', '22.6b0')
> return code: 1
> expected return code: 0
> stdout: (none)
> stderr:
>     error: pathspec '22.6b0' did not match any file(s) known to git
>     
> Check the log at /Users/razr/.cache/pre-commit/pre-commit.log
> [26/01 4:42:38] ➜  sort_contibutors git:(python-black-formatted) ✗ git status
> On branch python-black-formatted
> Your branch is up to date with 'origin/python-black-formatted'.
> 
> Changes to be committed:
>   (use "git restore --staged <file>..." to unstage)
> 	modified:   main.py
> ```

Updated in the next commit

File: .github/workflows/black.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: Black
+
+on: [push, pull_request]
```
Comment (commjoen) [inline .github/workflows/black.yml:3]: can you please remove this file :) ?

Comment (za) [reply .github/workflows/black.yml]: > can you please remove this file :) ? After that we should be good to merge and complete!

Removed.

**Review by commjoen — APPROVED** [review]
Thank you!

**Pull request merged**